### DISCUSSION TITLE
can't try spdlog

### DISCUSSION THREAD

**Discussion by CarlosLagarto** [SpriteOvO/spdlog-rs#11] [Q&A]
Hi,  I am studying the exchange of my current log machinery by spdlog-rs.

Can't move forward as something isn't working :-)

I am trying with

[dependencies]
---other stuff ---
spdlog-rs = {version = "0.2.3", default-features = false, features = ["flexible-string", "multi-thread"]}

![image](https://user-images.githubusercontent.com/95357309/168008403-c9a38dc5-ab72-490a-ac32-b4f747316bb7.png)

**Comment by SpriteOvO** [comment]
Asynchronous related stuffs (struct `AsyncSink`, feature `multi-thread`, etc.) are not released yet, we plan to release v0.3.0 with them when we finish `pattern-formatter`.

Currently you can only try non-asynchronous stuff, just remove `multi-thread` from `Cargo.toml`.

**Reply by CarlosLagarto** [reply]
Thanks for the reply.

Nevertheless I've pulled the code in the 2.4.0 version (just to have the thing running) and in that way it is working. 
I'll make some tests (from a user perspective :-) ) and let you know if something funny happens.
